src/database/migrations/0010-alter-application_forms-external_id.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE application_forms ADD COLUMN external_id VARCHAR;
+
+COMMENT ON COLUMN application_forms.external_id is 'An ID for this application form used by an external system.';
```

Comment:
This comment does not add any new information: we know it's for an application form because of the table, and we know it's an external ID because of the column. "System" is a new word, but that's essentially a stopword in this context.

I'm not against adding useful comments, but I think we'd need to add more detail to qualify. If we don't have any, let's just skip the comment!

```suggestion
```